cmd/influx/setup.go

```diff
@@ -236,54 +237,70 @@ You have entered:
 	}
 }
 
-var (
-	errPasswordIsNotMatch = fmt.Errorf("passwords do not match")
-	errPasswordIsTooShort = fmt.Errorf("passwords is too short")
-)
+func errSecretIsNotMatch(title string) error {
+	return fmt.Errorf(title + "s do not match")
+}
+
+const errTooShort = "too short"
+
+func errSecretIsTooShort(title string) error {
+	return &influxdb.Error{
+		Code: errTooShort,
```

Comment:
the code would be `influxdb.UnprocessableEntity` here, `too short` isn't a domain eror code. The message explains how it is out of range.  Let's remove the errTooShort code here